src/applications/vaos/tests/services/appointment/index.v2.unit.spec.js

```diff
@@ -730,4 +733,407 @@ describe('VAOS Appointment service', () => {
       expect(differences).to.be.empty;
     });
   });
+
+  describe('getAppointmentRequests', () => {
+    beforeEach(() => mockFetch());
+    it('should return matching v0 and v2 data for a VA appointment request', async () => {
+      // Given VA appointment request data
+      const data = {
+        id: '1234',
+        start: moment()
+          .add(3, 'days')
+          .format(),
+        email: 'test@va.gov',
+        phone: '2125551212',
+        kind: 'clinic',
+        locationId: '552GA',
+        clinic: '5544',
```

Comment:
Requests won't have clinics, you could remove this and I think it gets rid of some differences